electronic_mail.py

```diff
@@ -80,11 +87,13 @@ def _send_mail(cls, mails, smtp_server=None):
             mail_draft_mailbox = (mail.template.draft_mailbox or draft_mailbox
                 if mail.template else draft_mailbox)
 
-            if not mail_smtp_server:
+            if not mail_smtp_server or not mail_draft_mailbox:
```

Comment:
Better have two separate messages, one for mail_smtp_server and another for mail_draft_mailbox.

However, I don't really see the need for the 'mail_draft_mailbox' given that the field is required in `electronic.mail.template` model.

Reply:
Yes, but.... "send_mail" method is in electronic.mail model. You could have emails that not from a template. 

In case send btn:
"mailbox" and "server" from template or default values.

Reply:
You're right!